sources/dynamodb/schema.go

```diff
@@ -192,11 +192,30 @@ func (isi InfoSchemaImpl) ProcessData(conv *internal.Conv, srcTable string, srcS
 	return nil
 }
 
+// StartChangeDataCapture initializes the DynamoDB Streams for the source database. It
+// returns the latestStreamArn for all tables in the source database.
 func (isi InfoSchemaImpl) StartChangeDataCapture(ctx context.Context, conv *internal.Conv) (map[string]interface{}, error) {
-	return nil, nil
+	fmt.Println("Starting DynamoDB Streams initialization...")
+
+	LatestStreamArn := make(map[string]interface{})
+	orderTableNames := ddl.OrderTables(conv.SpSchema)
+
+	for _, spannerTable := range orderTableNames {
+		srcTable, _ := internal.GetSourceTable(conv, spannerTable)
+		streamArn, err := NewDynamoDBStream(isi.DynamoClient, srcTable)
+		if err != nil {
+			conv.Unexpected(fmt.Sprintf("Couldn't initialize DynamoDB Stream for table %s: %s", srcTable, err))
+			continue
```

Comment:
Yes, I agree that partial failures during table stream initialisation should not lead to failure of the whole command. 
I think that these failures need to be captured in the report so that the user is aware that `X out of Y` of all table streams were processed.

Reply:
Definitely, makes sense. We can add this information while making changes to the report file.